sdk/src/feature_set.rs

```diff
@@ -369,10 +369,15 @@ pub mod fix_recent_blockhashes {
 pub mod update_rewards_from_cached_accounts {
     solana_sdk::declare_id!("28s7i3htzhahXQKqmS2ExzbEoUypg9krwvtK2M9UWXh9");
 }
+
 pub mod enable_partitioned_epoch_reward {
     solana_sdk::declare_id!("9bn2vTJUsUcnpiZWbu2woSKtTGW3ErZC9ERv88SDqQjK");
 }
 
+pub mod migrate_epoch_stakes_snapshot_field {
+    solana_sdk::declare_id!("11111111111111111111111111111111");
+}
```

Comment:
Yeah, that's the way I was intending to use the feature gate. Do you have another suggestion? The way `epoch_rewards_status` was added a year ago also indirectly relied on an feature gate activation so that was why I went this route.

FWIW feature gates are not exclusively used for consensus related changes. To me, a feature gate is a good way to deal with the problem of wanting to change serialization at a point in time after the cluster is running a software version that can deserialize the new change.